www/components/BuiltWithSupabase/index.tsx

```diff
@@ -16,7 +16,7 @@ const BuiltExamples = () => {
       <div className="text-center">
         <Typography.Title level={2}>What can you build with supabase?</Typography.Title>
         <Typography.Text>
-          <p className="lg:text-lg">There's many example apps and starter projects to get going</p>
+          <p className="lg:text-lg">There're many example apps and starter projects to get going</p>
```

Comment:
Nice one. Perhaps we should remove the conjunction here for readability. "There're" is a bit awkward

Reply:
Yes, I think so.